package/yast2-crowbar.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Sat Apr  6 20:57:02 UTC 2019 - vuntz@suse.com
+
+- Update for changes to admin credentials in crowbar
+- Fix widgets being enabled due to wrong conditions
```

Comment:
Could you at least write "network widgets" so it is more specific? (Also the commit message please)

Reply:
done